src/Microsoft.Health.Fhir.Liquid.Converter/Filters/GeneralFilters.cs

```diff
@@ -73,6 +74,17 @@ public static string GenerateUUID(string input)
             var guid = new byte[16];
             Array.Copy(hash, 0, guid, 0, 16);
             return new Guid(guid).ToString();
+
+        }
+
+        public static string ToHtml(string input)
```

Comment:
`ToHtml` might not a appropriate name here. Since the method is actually encoding the HTML-like strings, not convert some other format of strings to HTML format.

Reply:
> `ToHtml` might be not an appropriate name here. Since the method is actually encoding the HTML-like strings, not converting some other format of strings to HTML format.

Thanks, changed the name to `EncodeToHtmlEscapedString`.

Reply:
Thanks, changed the name to `EncodeToHtmlEscapedString`.

Reply:
Will it be more appropriate to put it in `StringFilters` instead of `GeneralFilters`? It belongs to string processing.

Reply:
Good idea,  moved it.